lib/cisco_node_utils/cmd_ref/limit_resource.yaml

```diff
@@ -0,0 +1,10 @@
+# limit_resource
+---
+vdc:
+  _exclude: [/N3K/, /N9K/]
+  config_get: "show vdc current-vdc"
+  config_get_token: '/^Current vdc is.*\-\s+(\S+)/'
+
+vxlan:
+  _exclude: [/N3K/, /N9K/]
+  config_set: ["terminal dont-ask", "vdc %s", "limit-resource module-type f3", "end"]
```

Comment:
This is not actually code that I wrote but simply moved to a new file.  But agreed, will remove the end.